src/App.js

```diff
@@ -16,6 +16,13 @@ function App() {
   const { status } = useAuth();
   const isLoggedIn = status === AuthStatus.LoggedIn;
 
+  const ProtectedRoute = ({ children }) => {
+    if (!isLoggedIn) {
+      return <Navigate to="/signIn" />;
+    }
+    return children;
+  };
+
```

Comment:
Sorry this had slipped my mind when writing the ticket; but I belatedly realized we could have also done this with the `RequireAuth` component provided by `react-auth-context` but this works well as well; feel free to keep this or experiment with `RequireAuth` (though using RequireAuth may require some further changes to the auth settings in `index.js` so 🤷 )

https://github.com/akosasante/react-auth-context#:~:text=Alternatively%2C%20use%20the%20RequireAuth%20component%20in%20your%20router%20to%20wrap%20your%20component%20that%20should%20only%20be%20viewed%20if%20logged%2Din%3A

Reply:
I'm trying to do this option, but I think I'm going wrong